### DISCUSSION TITLE
Unable to install & run SDR++ / SDRPP on Pop_os

### DISCUSSION THREAD

**Discussion by Screak42** [AlexandreRouma/SDRPlusPlus#932] [General]
Hey!
Has anyone successfully installed SDR++ on Pop_OS? 
I had it installed on a few other distros and a mac without issues, but on pop I just can't get it to run.

I'm running into issues with libvolk which I'm unable to solve

```
dpkg: dependency problems prevent configuration of sdrpp:
sdrpp depends on libvolk1-dev; however:
Package libvolk1-dev is not installed.
```

I think I should be ok with an other version (v1 I can't get via apt)

```
libvolk2-bin is already the newest version (2.5.1-1).
libvolk2-dev is already the newest version (2.5.1-1).
```

Has anyone had success with this?

**Comment by AlexandreRouma** [comment]
wrong SDR++ deb was used

**Reply by Screak42** [reply]
Hey! Thanks for your reply :)

I did actually try all versions as a really dumb approach, but I run into either the above error, or a shared object file error I can't get around.

`sdrpp: error while loading shared libraries: libvolk.so.2.4: cannot open shared object file: No such file or directory`

As I only have 2.5 available to install via apt, I was hoping I can still use it - but to no avail.

I've made sure `/etc/ld.so.conf` contains the paths to my availabe libvolk's and ran `ldconfig`.

**Reply by illtellyoulater** [reply]
> wrong SDR++ deb was used

@AlexandreRouma could you please make clear which package a Pop OS user should pick?

**Reply by e-m-m-a** [reply]
@AlexandreRouma  Oh me too, I'm on a failed install with the sdrpp_debian_bullseye_amd64.deb nightly

**Comment by regenerous** [comment]
I would also like to know which package I should be using for Pop_Os

**Comment by lawsonjt** [comment]
I've got it working on Pop_Os by doing the following:

Install dependencies:
`sudo apt install build-essential cmake git libfftw3-dev libglfw3-dev libvolk2-dev libsoapysdr-dev libairspyhf-dev libairspy-dev libhackrf-dev libiio-dev libad9361-dev librtlsdr-dev libbladerf-dev liblimesuite-dev`

Clone Repo:
`https://github.com/AlexandreRouma/SDRPlusPlus.git`

Build SDR++:
```
cd SDRPlusPlus
mkdir build
cd build
```

Compile SDR++ (install cmake if you don't already have it):
```
cmake ..
make -j$(nproc)
```

Run it!
`./sdrpp`
